new/aws-ec2-manager-blueprint-inputs.yaml

```diff
@@ -120,4 +120,38 @@ ec2_region_name: ''
 #############################
 # InfluxDB Inputs
 #############################
-#influxdb_endpoint_ip: ''
\ No newline at end of file
+#influxdb_endpoint_ip: ''
+
+
+###########################
+# Offline Resources Upload
+###########################
+# You can configure a set of resources to upload at bootstrap. This resources
+# will reside on the manager and enable offline deployment. dsl_resources
+# should contain any resource needed in the parsing process (i.e. plugin.yaml files),
+# and any plugin archive should be compiled through the designated wagon tool,
+# and the path should be passed to plugin_resources. Any resource you're
+# blueprint might need, could be upload trough this mechanism.
+#dsl_resources:
+#  - {'source_path': '/cfy/cloudify/plugins/fabric-plugin/plugin.yaml', 'destination_path': '/fabric-plugin/1.3rc1/plugin.yaml'}
+#  - {'source_path': '/cfy/cloudify/plugins/script-plugin/plugin.yaml', 'destination_path': '/script-plugin/1.3rc1/plugin.yaml'}
+#  - {'source_path': '/cfy/cloudify/plugins/diamond-plugin/plugin.yaml', 'destination_path': '/diamond-plugin/1.3rc1/plugin.yaml'}
+#  - {'source_path': '/cfy/cloudify/plugins/aws-plugin/plugin.yaml', 'destination_path': '/aws-plugin/1.3rc1/plugin.yaml'}
+#  - {'source_path': '/cfy/cloudify/types/types.yaml', 'destination_path': '/cloudify/3.3rc1/types.yaml'}
+#plugin_resources:
+#################
+# Diamond plugin
+#################
+#  - 'http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/3.3.0/m7-RELEASE/cloudify_diamond_plugin-1.3a7-py26-none-linux_x86_64-centos-Final.tar.gz'
```

Comment:
question: plugin.yaml files included in the plugin (e.g. diamond-plugin/plugin.yaml) still have to be uploaded separately as dsl_resources?